.changeset/green-buses-train.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/code-sdk': patch
+---
+
+Improved diagnostics and error guidance when deployed tenant credential resolution fails closed.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)